addons/misra.py

```diff
@@ -1924,8 +1924,8 @@ def reportError(self, location, num1, num2):
                 return
             formattedMsg = cppcheckdata.reportError(args.template,
                                                     callstack=[(location.file, location.linenr)],
-                                                    severity='style',
-                                                    message = errmsg,
+                                                    severity= 'style',
+                                                    message = 'MISRA: ' + errmsg,
```

Comment:
if you are pedantic.. this might break our agreement with MISRA. We are not allowed to alter the message. But well.. the "MISRA" is not supposed to be read as the message.

Reply:
Better safe than sorry...

Reply:
yes. can you please put the addon in the message id instead?